### DISCUSSION TITLE
Crash while editing a workflow

### DISCUSSION THREAD

**Discussion by marcusrbrown** [community/community#59792] [Mobile]
I was editing a workflow in a repository I have collaborator access. The app crashed, dropping me back to the Homescreen.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.119.0 (102894307)
Device: iPhone15,2 (iOS 16.5.1)
</details>

**Comment by alcere** [comment]
Hi @marcusrbrown, thanks for reporting the issue, and sorry for the troubles.
I tried to reproduce the issue you described but I couldn't, do you mind sharing more details like the role of your account and the steps you followed? Thanks!

**Reply by marcusrbrown** [reply]
Of course. The workflow I was editing is [.github/workflows/github-app.yaml](https://github.com/fro-bot/.github/blob/main/.github/workflows/github-app.yaml). @fro-bot is a user account (I use it as my service account) where I have granted myself collaborator access. 

I have the fro-bot/.github repo saved as a Favorite, so I tapped it to get to the repo. I navigated to the file and used 'Edit file' from the menu. I began making changes to the file (adding new lines and text, deleting text, etc.), and the app crashed approx. 2-3 minutes while editing. The crash wasn't immediate.

**Reply by alcere** [reply]
Oh! I am so sorry to hear that and the data loss! I will create an issue and we will investigate the problem. 
Thanks again for the heads up!
